app/assets/stylesheets/card-input.scss

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file contains the styles required to use cards as radioboxes.
+ * Loosely based upon an example found here: https://jsfiddle.net/djibe89/at2cv9q1/.
+ */
+
+@import url(https://fonts.googleapis.com/icon?family=Material+Icons);
```

Comment:
Ik zal later de Bootstrap icons wel lokaal importeren, maar voor nu is dit prima.

Reply:
Ok!